ui/src/preferences/about-system/AboutSystem.tsx

```diff
@@ -45,15 +43,6 @@ export const AboutSystem = () => {
             <p>
               Version {gardenCharge?.version} ({hash})
             </p>
-            {systemBlocked && (
-              <p>
-                Aeon {aeon}{' '}
-                <span className="text-orange-500 mx-4 space-x-2">
-                  <span>&mdash;&gt;</span> <span>/</span> <span>&mdash;&gt;</span>
-                </span>{' '}
-                Aeon {nextAeon}
-              </p>
-            )}
```

Comment:
is there no replacement for this in the pike system?

Reply:
Here is the schema for Pike:
```ts
export interface Pike {
  /**
   * Hash of the desk, rendered as `@uv`
   *
   * @remarks
   * Equivalent to
   * ```hoon
   * .^(@uv %cz /=desk=)
   * ```
   */
  hash: string;
  sync: {
    /**
     * Source desk for this Pike
     */
    desk: string;
    /**
     * Source ship for this Pike
     */
    ship: string;
  } | null;
  /**
   *  {@link Weft}s associated with this Pike
   */
  wefts: Weft[];
  /**
   * how live is this pike?
   * 
   * live - app is running
   * held - app is not running, but is trying to run. this state can be entered
   * in two main ways: 
   *   - when installing an app but it hasn't finished downloading (or it did 
   *     but failed to install for some reason)
   *   - when user forced a kelvin upgrade by suspending desks.
   * dead - app is not running
   */
  zest: "live" | "dead" | "held";
}
```

And here's a Weft:
```ts
/**
 * A component's kelvin version
 */
export interface Weft {
  /**
   *  Name of the component
   *
   *  @remarks
   *  Usually %zuse, %hoon, or %lull
   */
  name: string;
  /**
   * Kelvin version
   *
   */
  kelvin: number;
}
```

Reply:
So looking at the new schema posted in previous comment, not seeing any `aeon` fields.